.github/workflows/inventory.yml

```diff
@@ -0,0 +1,69 @@
+name: Update Inventory
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '00 4 * * 1-5'
+
+jobs:
+  update-nodejs-inventory:
+    name: Update Node.js Engine Inventory
+    runs-on: pub-hk-ubuntu-22.04-small
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - id: install-rust-toolchain
```

Comment:
(The `id` attribute is only required if the step needs to be referenced from elsewhere, eg from contexts - so can be omitted if preferred)